src/app/hooks/useElements.ts

```diff
@@ -176,7 +238,7 @@ export const useElements = ({
     useEffect(() => {
         shouldResetCache() && resetCache();
         shouldSendRequest() && load();
-    }, [labelID, page, sort, filter]);
+    }, [labelID, page.page, sort, filter, address, from, to, keyword, begin, end, attachments, wildcard]);
```

Comment:
Not really sure to remember but I think I use the whole page object for a good reason.

By looking into it a bit, I would say it's for the `isExpectedLength` test which use all page properties.